### ISSUE TITLE
Add `*.cts, *.ctsx, *.mts, *.mtsx` extensions to `overrides.files` in `configs/recommended.js`

### ISSUE THREAD

**Issue by david-bojnansky** [sindresorhus/eslint-plugin-unicorn#1578]
The rules should be applied to files with new extensions: `.cts, .ctsx, .mts, .mtsx`.

```js
'use strict';
module.exports = {
	...require('./base.js'),
	rules: {

		...

		'unicorn/require-post-message-target-origin': 'error',
		'unicorn/string-content': 'off',
		'unicorn/template-indent': 'warn',
		'unicorn/throw-new-error': 'error',
		...require('./conflicting-rules.js').rules,
	},
	overrides: [
		{
			files: ['*.ts', '*.tsx', '*.cts', '*.ctsx', '*.mts', '*.mtsx'], // <------------------ HERE
			rules: {
				'unicorn/require-post-message-target-origin': 'off',
			},
		},
	],
};
```
